backend/src/tasks/helpers/simple-wappalyzer.ts

```diff
@@ -29,7 +29,7 @@ import { chain, mapValues } from 'lodash';
 import { JSDOM, VirtualConsole } from 'jsdom';
 import { Cookie } from 'tough-cookie';
 
-const data = require('./technologies.json');
+const data = { categories: {}, technologies: require('./technologies.json') };
```

Comment:
is this instead of the @colin-tim fix?

Reply:
No, it's just a temporary fix that allows the worker to build without errors for now. Let's merge this PR now and then replace it with @colin-tim 's fix when that PR is merged, because Colin's fix actually allows the wappalyzer scan to work as intended.